docs/meetings/dev/2024-12-17.md

```diff
@@ -0,0 +1,35 @@
+# Weekly meeting
+
+When: 12/24/24, 21:00 UTC.
+Present: Aapeli, David, Jesse, Nicole, Yannic, Gracee, Darren
+Scribe: Nicole
+
+# Notes
+
+* Aapeli talks about board meeting take-aways.
+    - Nicole joined the board.
+    - We're going to focus our identity on being a safe an active community.
+    - Ban on nudists
```

Comment:
Are we starting to see this a lot on our platform too? 😅

Reply:
Somewhat, can elaborate on Slack 😅